src/System.ComponentModel.Annotations/src/System/ComponentModel/DataAnnotations/DisplayFormatAttribute.cs

```diff
@@ -11,6 +13,9 @@ namespace System.ComponentModel.DataAnnotations
     [AttributeUsage(AttributeTargets.Property | AttributeTargets.Field, AllowMultiple = false)]
     public class DisplayFormatAttribute : Attribute
     {
+        private readonly LocalizableString nullDisplayText = new LocalizableString("NulLDisplayText");
+        private Type resourceType;
```

Comment:
Regarding underscores, I did follow these [guidelines](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/framework-design-guidelines-digest.md) but I will change for consistency with the rest.